esmvaltool/cmorizers/obs/cmorize_obs_e_obs.py

```diff
@@ -0,0 +1,98 @@
+"""ESMValTool CMORizer for CRU data.
+
+Tier
+    Tier 2: other freely-available dataset.
+
+Source
+    http://surfobs.climate.copernicus.eu/dataaccess/access_eobs.php#datafiles
+
+Last access
+    20200225
+
+Download and processing instructions
+    Download the ensemble mean files for:
+        TG TN TX RR PP
+    make cdo monmean over the files to the new name *.nc -> *_mon.nc
```

Comment:
The cmorizer should take care of every processing step.

Reply:
Yes it does. Thanks for checking.